lib/node_modules/@stdlib/strided/base/zmap/lib/ndarray.js

```diff
@@ -51,13 +51,7 @@
 * zmap( x.length, x, 1, 0, y, 1, 0, scale );
 *
 * var v = y.get( 0 );
-* // returns <Complex128>
-*
-* var re = real( v );
-* // returns 10.0
-*
-* var im = imag( v );
-* // returns 10.0
+* // returns <Complex128>[ 10.0, 10.0 ]
```

Comment:
@prajjwalbajpai You need to remove any applicable import statements in this and other examples in this file.